urumanifest/constants.py

```diff
@@ -153,4 +158,6 @@ class PyToolsResultCodes(enum.IntEnum):
     "plasma-windows-x64-external-release": "external64",
     "plasma-windows-x86-internal-release": "internal",
     "plasma-windows-x64-internal-release": "internal64",
+    "plasma-macos-x64-internal-release": "macInternal",
+    "plasma-macos-x64-external-release": "macExternal",
```

Comment:
Just noting that these are Intel x86_64 binaries (which is probably the best option for the moment, but they are not universal because they do not include arm64)

Reply:
Yeah - that still needs to be done on the CI side. For now, I'm kind of assuming whatever the platforms should be is what is provided by CI.